documentation/src/main/asciidoc/topics/ref_jgroups_system_properties.adoc

```diff
@@ -1,11 +1,6 @@
 [id='jgroups_system_props-{context}']
-= System Properties for Default JGroups Stacks
-Use system properties with default JGroups stacks.
-
-.Common Properties
-
-The following system properties apply to all JGroups stacks, including
-`default-jgroups-udp.xml` and `default-jgroups-tcp.xml`:
```

Comment:
there is one more property I added with JGroups 4.2.9:
`jgroups.thread_dumps_threshold`: "The number of times a thread pool needs to be full before a thread dump is logged"

Reply:
@pruivo is the default for `jgroups.thread_dumps_threshold` 10000?

Reply:
yes